st2common/st2common/util/misc.py

```diff
@@ -56,9 +56,26 @@ def compare_path_file_name(file_path_a, file_path_b):
     return file_name_a < file_name_b
 
 
-def strip_last_newline_char(input_str):
+def strip_shell_chars(input_str):
     """
-    Strips the last char if its newline.
+    Strips the last '\r' or '\n' or '\r\n' string at the end of
+    the input string. This is typically used to strip ``stdout``
+    and ``stderr`` streams of those characters.
+
+    :param input_str: Input string to be stripped.
+    :type input_str: ``str``
+
+    :rtype: ``str``
+    """
+    stripped_str = rstrip_last_char(input_str, '\n')
+    stripped_str = rstrip_last_char(stripped_str, '\r')
+    return stripped_str
+
+
+def rstrip_last_char(input_str, char_to_strip):
```

Comment:
It looks like the second argument should be called `word/str_to_strip` since it supports stripping multiple characters and that's also what you do in some of the tests :P